components/Input/index.html

```diff
@@ -78,10 +78,78 @@ <h3 class="sidebar-head">Components</h3>
 					</li>
 				</ul>
 			</div>
-			<div class="component">
-				<h2>Input</h2>
+			<div class="flex-center-component">
+				<div class="component">
+					<h1 class="component-header">Input</h1>
+					<p class="component-text">
+						Use the Input component to collect user data such as emails and
+						passwords. Use the class
+						<span class="highlight">input</span> for input component. You can
+						refer to the code given below.
+					</p>
+
+					<h2 class="component-sub-header">Input types</h2>
+
+					<div class="component-display vertical">
+						<input type="text" class="input" placeholder="Enter Text..." />
+						<input type="number" class="input" placeholder="Enter Number..." />
```

Comment:
you should use input with label tag

Reply:
I assumed not all input fields require a label. I'll add this and more input types later on